RR.App/Controllers/HRIS/AuthenticationController.cs

```diff
@@ -15,8 +16,8 @@ public AuthenticationController(IAuthService authService)
         _authService = authService;
     }
 
-    [AllowAnonymous]
-    [ApiExplorerSettings(IgnoreApi = true)]
+    //[AllowAnonymous]
+    //[ApiExplorerSettings(IgnoreApi = true)]
```

Comment:
Refactor: Re-add AllowAnon, as it gives the message to the developer that this endpoint should allow foreign requests. The IgnoreApi can be removed as then it would be added to the api documentation.

Reply:
done